### PR TITLE
Add ability to set a dark/light class on the preview iframe

### PR DESCRIPTION
**Pull request by jpzwarte**
See #101.

### CODE REVIEW

**Comment by jpzwarte** [comment]
This is my first attempt, haven't even tried to run it :)

File: src/Tool.tsx [hunk 1]
```diff
@@ -38,6 +42,27 @@ const update = (newStore: DarkModeStore) => {
   window.localStorage.setItem(STORAGE_KEY, JSON.stringify(newStore));
 };
 
+/** Update the preview iframe class */
+const updatePreview = (newStore: DarkModeStore) => {
+  const iframe = document.getElementById('storybook-preview-iframe') as HTMLIFrameElement;
```
Comment (hipstersmoothie) [inline src/Tool.tsx:47]:
```suggestion
  const iframe = document.getElementById<HTMLIFrameElement>('storybook-preview-iframe');
```

**Comment by jpzwarte** [comment]
How do you test it? `yarn build` and then link it to a repo that uses storybook?

**Comment by hipstersmoothie** [comment]
```sh
# in dark mode addon
yarn build
yarn link
# in root of your storybook project
yarn link storybook-dark-mode
```

And that should work

**Comment by hipstersmoothie** [comment]
Or if you rebase and fix the build a canary "test" version will be published for the PR

**Comment by hipstersmoothie** [comment]
:rocket: PR was released in [`v0.5.0`](https://github.com/hipstersmoothie/storybook-dark-mode/releases/tag/v0.5.0) :rocket:

**Comment by hipstersmoothie** [comment]
Ah it slipped my mind what part of the UI you were adding this class to. I see now that it is the preview iframe. I actually will be making this an optional feature. Before this PR there has been a distinct separation between the storybook UIs dark mode and that of your components. The hooks were opt in way to style you components based on the dark mode of the UI. With this change that is no longer true and the plugin will try to render dark mode for the components.

My incoming PR will:

1. make this feature an option
2. add an option to add this class to the root of the storybook UI (what I wanted and thought was being contributed)

**Pull request merged**